docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -32,75 +42,113 @@ components:
 - **Layer 1 fee** - The L1 fee is the cost to publish your L2 transaction onto Ethereum and can 
     vary based on the blob fee market.
 
-## Estimating transaction costs
+To read more about how gas works on Linea, see [_Gas on Linea_](./gas-on-linea.mdx).
 
-Use the [`linea_estimateGas`](../../reference/api/linea-estimategas.mdx) API method to estimate the 
-gas cost for sending a transaction. The method returns the recommended gas limit, the base fee per 
-gas, and the priority fee per gas.
+## Estimating transaction costs
 
-The `linea_estimateGas` endpoint is currently unavailable. For more information, see 
-our [reference page](../../reference/api/linea-estimategas.mdx).
+:::info
+The best method for estimating gas on Linea, `linea_estimateGas`, is currently unavailable, and will
```

Comment:
It is not a great perception to say that the best method is not available yet imo. Perhaps we say: A new API for estimating gas, linea_estimateGas, will be activated soon.